pkg/machine/machinesetlist.go

```diff
@@ -0,0 +1,44 @@
+package machine
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	machinev1beta1 "github.com/openshift/api/machine/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListWorkerMachineSets returns the list of MachineSets in a namespace on a cluster.
+func ListWorkerMachineSets(
+	apiClient *clients.Settings,
+	namespace string) (*machinev1beta1.MachineSetList, error) {
```

Comment:
This should return ([]*MachineSetBuilder, error)

Reply:
Done